be/tsconfig.json

```diff
@@ -6,16 +6,20 @@
     "moduleResolution": "node",
     "esModuleInterop": true,
     "strict": true,
+    "strictPropertyInitialization": false,
+    "strictNullChecks": false,
     "sourceMap": false,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "rootDir": "./",
+    "rootDir": "./src",
     "baseUrl": "./",
     "types": ["jest"],
     "outDir": "dist/"
   },
+  "include": ["src/**/*.ts"],
   "exclude": [
     "node_modules/",
-    "jest.config.ts"
+    "jest.config.ts",
+    "src/generated/"
```

Comment:
Why should this be excluded? If we generate `.ts`, it needs to be compiled...

Reply:
The `jest.config.ts` is just there for configuring jest to run all tests. This file is not needed in the development build